pgmpy/models/NaiveBayes.py

```diff
@@ -141,9 +141,8 @@ def active_trail_nodes(self, start, observed=None):
 
     def local_independencies(self, variables):
         """
-        Returns a list of independencies objects containing the local independencies
-        of each of the variables. If local independencies does not exist for a variable
-        it gives a None for that variable.
+        Returns a independencies object containing the local independencies
```

Comment:
@raghavg7796 I would suggest you to use something like this: `Returns an instance of Independencies`. Feels technically more correct.